### ISSUE TITLE
Defauls collision mask is offset 1px

### ISSUE THREAD

**Issue by Wend1go** [4ian/GD#519]
## Describe the bug
The default collisiom mask of a sprite is offset by 1px on the Y-axis.
If you create a custom collisiom mask and add 1 to the Y axis everything looks as expected.
![image](https://user-images.githubusercontent.com/7413059/41229503-82b9dd9e-6d7c-11e8-94c0-0137fce304a3.png)

## Other details
GDevelop 5.0.0-beta39

**Comment by 4ian** [comment]
Seems that there is an offset indeed. I think it's just the rendering of the mask in the editor that is faulty.

**Comment by Wend1go** [comment]
It is also in the preview. the picture in the upper left corner was taken form the preview with the player standing on the black floor.

**Comment by 4ian** [comment]
Yes, I do think that the picture in the upper left corner taking from the preview is right, as there is one pixel between the character and the bottom of the image (as shown by the black border around the character in the editor), is that right? (the green colored area in my image below:
So the thing is that the red collision mask should have on extra pixel height?

<img width="682" alt="untitled-1" src="https://user-images.githubusercontent.com/1280130/41231274-78f03864-6d7a-11e8-9001-5db0fcf8625f.png">

**Comment by Wend1go** [comment]
The whole sprite seams to be shifted up by one pixel as the red box overlaps the black border.
![image](https://user-images.githubusercontent.com/7413059/41233310-620060c0-6d89-11e8-86e5-afe256a73b63.png)
The original sprite (checked in Piskel and Gimp) has 2 pixels above the head and the feet are at the very bottom of the image.
